lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart

```diff
@@ -260,9 +260,11 @@ class HtmlViewEmbedder {
     DomElement platformView,
     DomElement headClipView,
   ) {
-    int indexInFlutterView = -1;
-    if (headClipView.parent != null) {
-      indexInFlutterView = skiaSceneHost!.children.indexOf(headClipView);
```

Comment:
This is actually safe for now, but nice to get rid of if we decide we want to move off using `List` for `children`.